solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -1255,7 +1255,11 @@ private void createEphemeralLiveNode() throws KeeperException, InterruptedExcept
                     zkClient.getZkACLProvider().getACLsToAdd(nodePath),
                     CreateMode.EPHEMERAL)));
 
-    zkClient.multi(ops, true);
+    try {
+      zkClient.multi(ops, true);
```

Comment:
@noblepaul it is multi ops. Does it throw all exception?
1. all ops applied?
2. one fails and it throws exception?
Just want to make sure all ops applied and no other exception

Reply:
True.

I think multi-op is not suitable here. So, I just changed to to single op